deploy/astra-connector/clusterrole.yaml

```diff
@@ -19,5 +19,5 @@ rules:
   resources: ["tridentversions", "tridentorchestrators"]
   verbs: ["watch", "list", "get"]
 - apiGroups: ["astra.netapp.io"]
-  resources: ["applications", "appmirrorrelationships", "appmirrorupdates", "appvaults", "autosupportbundles", "backups", "backupinplacerestores", "backuprestores", "exechooks", "exechooksruns", "pvccopies", "pvcerases", "resourcebackups", "resourcedeletes", "resourcerestores", "resourcesummaryuploads", "resticvolumebackups", "resticvolumerestores", "schedules", "snapshotinplacerestores", "snapshotrestores", "snapshots", "astraconnectors"]
-  verbs: ["watch", "list", "get"]
\ No newline at end of file
+  resources: ["applications", "appmirrorrelationships", "appmirrorupdates", "appvaults", "autosupportbundles", "backups", "backupinplacerestores", "backuprestores", "exechooks", "exechooksruns", "pvccopies", "pvcerases", "resourcebackups", "resourcedeletes", "resourcerestores", "resourcesummaryuploads", "resticvolumebackups", "resticvolumerestores", "schedules", "snapshotinplacerestores", "snapshotrestores", "snapshots", "astraconnectors", "shutdownsnapshots", "autosupportbundleschedules", "kopiavolumebackups"]
```

Comment:
what about kopiavolumerestores ?

Reply:
didn't know that was a thing. I can add it